app/src/main/java/com/nutomic/syncthingandroid/service/EventProcessor.java

```diff
@@ -170,6 +181,17 @@ public void onEvent(Event event) {
             case "Ping":
                 // Ignored.
                 break;
+            case "Starting":
+            case "StateChanged":
+            case "StartupComplete":
+            case "DeviceConnected":
+            case "FolderSummary":
+            case "RemoteIndexUpdated":
+            case "LocalIndexUpdated":
+            case "ItemStarted":
+            case "FolderPaused":
+            case "FolderScanProgress":
+            case "LoginAttempt":
```

Comment:
I would break after this. These states are handled, we know about them but choose to do nothing about them.
You also might want to update completion on IndexUpdated events.

Reply:
And also folder summary events for local completion

Reply:
Added to my todo list.